busypie/func.py

```diff
@@ -27,5 +27,5 @@ def _is_a_lambda(func):
 
 def _content_of(lambda_func):
     source_line = inspect.getsource(lambda_func)
-    r = re.search(r'lambda:\s*(.+)\s*\)', source_line)
+    r = re.search(r'lambda[^:]*:\s*(.+)\s*\)', source_line)
```

Comment:
This fix should be covered in test_description as it's only relevant to the description

Reply:
I'm actually not even sure now why this needs to strip `lambda[^*]:\s` from the description? What was your intent in this?  We got around this bug by using `with_description` but it was incredibly confusing how to fix it at first without digging into the code itself.   Why do you not want to show the entire lambda fn in the description?

Reply:
I've moved the test to `test_description.py` but I still think it would be good to know why stripping the lambda keyword off is important in reporting whats going on

Reply:
The intent was to show the the actual condition, making the "lambda" keyword redundant. Does this makes sense?

Reply:
Yeah it does now, and only triggers when the condition fails